src/main/kotlin/convergence/discord/frat/FratCommands.kt

```diff
@@ -83,6 +112,16 @@ fun registerFratCommands() {
             "brotherbynickname (nickname)"
         )
     )
+    registerCommand(
+        Command(
+            DiscordProtocol,
+            "brothergetline",
+            listOf(ArgumentSpec("Name", ArgumentType.STRING)),
+            { args -> brotherLine(args) { it.firstName + " " + it.lastName } },
```

Comment:
You don't need to pass this as a lambda, you can just hardcode it, I did it as a lambda for brother info because the three commands (by roster, by name, by nickname) each use different criteria and I wanted to reuse the function for each one